frontend/packages/component-library/src/video/Video.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import moduleStyles from './video.module.scss';
+import LinkButton from '../button/LinkButton';
+
+export interface VideoProps extends HTMLAttributes<HTMLElement> {
+  /** Video Title */
+  videoTitle?: string;
+  /** Video YouTube ID */
+  youTubeId?: string;
+  /** Video fallback */
+  videoFallback?: string;
+  /** Show caption */
+  showCaption?: boolean;
+  /** Video custom className */
+  className?: string;
+}
+
+/**
+ * ### Production-ready Checklist:
+ * * (✘) implementation of component approved by design team;
+ * * (✘) has storybook, covered with stories and documentation;
+ * * (✘) has tests: test every prop, every state and every interaction that's js related;
+ * * (see ./__tests__/Video.test.tsx)
+ * * (✘) passes accessibility checks;
+ *
+ * ###  Status: ```WIP```
+ *
+ * Design System: Video Component.
+ */
+
+export const Video: React.FC<VideoProps> = ({
```

Comment:
This component is pretty specific to YouTube, would it make sense to call this `YouTubeVideo` component?

Reply:
It'll be less so once I add the fallback option 🙌